### PR TITLE
Add-Debian-11-to-QA-testsuit

### PR DESCRIPTION
**Pull request by Shirocco88**
## What does this PR change?
Add-Debian-11-to-QA-testsuit

**add description**
this PR was splitted from PR https://github.com/uyuni-project/uyuni/pull/4180

## Test coverage
- Unit tests were added

- [ ] **DONE**

## Links

- Prepare test.env to support Ubuntu11 client https://github.com/SUSE/spacewalk/issues/15781
- Port to SUSE Manager https://github.com/SUSE/spacewalk/issues/15782

## Changelogs

If you don't need a changelog check, please mark this checkbox:

- [x] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

### CODE REVIEW

**Comment by srbarrios** [comment]
Awesome! Thank you!
It requires some more work, as we need to prepare new domain names, macs, and IPs in our DHCP/DNS.
Also, we need to add new VMs in our terraform file. But this PR looks good, and as we have Cucumber tags for them, using an environment variable to check if that's deployed or not, will not break anything.

So for me, we can merge, approving and creating a card in our qa squad backlog for the rest.

**Review by srbarrios — APPROVED** [review]

**Comment by Shirocco88** [comment]
Hint:
at the moment the run_set for Debian-11 is disabled in testsuite/run_sets/build_validation_init_clients.yml

**Review by calancha — APPROVED** [review]
I agree with what Oscar have said.
Thank you @Shirocco88 !

File: testsuite/run_sets/build_validation_init_clients.yml
Comment (Bischoff) [inline testsuite/run_sets/build_validation_init_clients.yml:42]: i would just uncomment this

but we can do it in a separate PR

**Review by Bischoff — APPROVED** [review]
see little nitpick

**Comment by Bischoff** [comment]
Hi @Shirocco88 . Sorry it took so much time.

I would like to merge your work. However,
* I'd like to uncomment 2 lines (see my review)
* and, more important, it would need some rebasing and resolving conflicts

Can you do that ? If you don't have the time, could you leave me push access to your repository, so I do it myself ?

Thanks in advance.

**Comment by juliogonzalez** [comment]
I removed the label until this is ready to be merged. Do not the label unless a PR is ready for merging :-D

**Review by Bischoff — APPROVED** [review]
Fixed the conflicts and removed comments.

However I created 7 commits :-( . If someone can squash...

**Pull request merged**
